packages/eslint-plugin/tests/rules/sort-ngmodule-metadata-arrays/cases.ts

```diff
@@ -215,17 +216,271 @@ export const invalid = [
     @NgModule({
       providers: [
         AProvider,
+        bProvider,
+        cProvider,
+        DProvider,
+        ~~~~~~~~~
```

Comment:
This change is due to the fact that the fixer now, puts everything that isn't a `Identifier` at the end, to avoid conflicts and simplify the implementation.